api-test-extension/src/tests/content_settings.js

```diff
@@ -66,9 +66,6 @@ var content_settings_test = new TestSet()
     .require("[Property Exists] cookies", methodExists(chrome.contentSettings, 'cookies'), { hideOnSuccess: true })
     .require("[Property Set] cookies", methodCall(chrome.contentSettings.cookies, 'set', detailsBlock, () => {}))
 
-    .require("[Property Exists] images", methodExists(chrome.contentSettings, 'images'), { hideOnSuccess: true })
-    .require("[Property Set] images", methodCall(chrome.contentSettings.images, 'set', detailsBlock, () => {}))
```

Comment:
Tests are also designed to show which API is unsupported, we can't just remove them. There is the way to display that test fail is expected:
`.suggest("[Property Set] images {Unsupported on Android}", () => "Test disabled")`